user-interface/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleContentComponent.java

```diff
@@ -75,7 +93,6 @@ private boolean isComponentInContent(Component component) {
 
   private void displayComponentInContent(Component component) {
     if (!isComponentInContent(component)) {
-      this.removeAll();
```

Comment:
what was this about?

Reply:
This was a shorthand to ensure that the same component is not added multiple times. It's now replaced with the specific check on if the added component was already added